tests/unit_tests/test_wave_06.py

```diff
@@ -76,16 +76,22 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    # raise Exception("Complete this test according to comments below.")
     # Assertions should check:
     # - That the results is truthy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert result
+    assert len(jesse.inventory) == len(tai.inventory)
```

Comment:
Instead of looking at if the lists are the same as each other, it would be good to check that each list is the length you expect it to be. If you accidentally added items to both inventories without removing anything, this would still pass even though that shouldn't happen.